src/Cli/Utils.cs

```diff
@@ -324,9 +330,9 @@ public static bool TryParseMappingDictionary(IEnumerable<string> mappingList, ou
 
             // If restPath is null, it implies we are dealing with cosmosdb_nosql,
             // which only supports graphql.
-            if (restPath is not null)
+            if (restPath is not null && restBaseRoute is not null)
             {
-                defaultGlobalSettings.Add(GlobalSettingsType.Rest, new RestGlobalSettings(Enabled: restEnabled, Path: restPath));
+                defaultGlobalSettings.Add(GlobalSettingsType.Rest, new RestGlobalSettings(Enabled: restEnabled, Path: restPath, BaseRoute: restBaseRoute));
```

Comment:
We'll have a merge conflict with #1402 as the structure of the config we store this in has changed. It'll be a new property on `RestRuntimeOptions` and we'll have to set it appropriately there.